app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -507,9 +520,27 @@ class BrowserTabViewModel(
             statisticsUpdater.refreshSearchRetentionAtb()
         }
 
+        updateLoadingStatePrivacy()
+        domain?.let { updateWhitelistedState(domain) }
         registerSiteVisit()
     }
 
+    private fun updateLoadingStatePrivacy() {
+        viewModelScope.launch(dispatchers.io()) {
+            site?.domain?.let {
+                loadingViewState.postValue(currentLoadingViewState().copy(privacyOn = !userWhitelistDao.contains(it)))
```

Comment:
Question about this. Are we only disabling the trackers animation/dax dialogs if the user has a specific website in the whitelist? What happens if they disable privacy protection overall or is that not possible anymore?

Reply:
@marcosholgado there is no longer a global switch.